src/App.js

```diff
@@ -12,30 +11,43 @@ class App extends Component {
     this.state = {
       movies: [],
       showMain: true,
+      error: '',
+      displayMovie: {},
+      trailers: [],
     }
   }
 
   componentDidMount = () => {
   Promise.resolve(fetchData('movies'))
   .then(data => this.setState( { movies: data.movies } ))
```

Comment:
Do you NEED all of the data coming back from this fetch call? If you are not using every piece of data, you should clean the data first before adding it to state.

Reply:
Could you please expand your comment here Kayla? Shouldn't we have all the properties we will need on the state declared by default?